src/inference/tests/functional/shared_object_loader_test.cpp

```diff
@@ -1,68 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include <file_utils.h>
-#include <gtest/gtest.h>
-
-#include <cpp/ie_plugin.hpp>
-
-#include "common_test_utils/file_utils.hpp"
-#include "openvino/util/shared_object.hpp"
-
-using namespace std;
-using namespace InferenceEngine;
-using namespace InferenceEngine::details;
-
-class SharedObjectLoaderTests : public ::testing::Test {
```

Comment:
Why did you remove this file?

Reply:
Because same tests are presented in [src\inference\tests\functional\ov_shared_object_test.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/inference/tests/functional/ov_shared_object_test.cpp)